scripts/src/main/resources/scripts/command/java

```diff
@@ -91,8 +87,214 @@ function doCreate() {
+if [[ -d "${artifact}" ]]
+then 
+  doFail "A project: ${artifact} already exists, make sure to delete the existing project or rename your new project and run the comment again"
+fi
+
+if [[ -e "${DEVON_IDE_HOME}/updates/quarkus.zip" ]]
+then 
+  rm "${DEVON_IDE_HOME}/updates/quarkus.zip"
+fi 
+doDownload "${url}" "quarkus.zip" "${DEVON_IDE_HOME}/updates"
+
+doExtract "${DEVON_IDE_HOME}/updates/quarkus.zip"
+rm "${DEVON_IDE_HOME}/updates/quarkus.zip"
+
+local groupIdFolders="${group//[.]/"/"}"
+
+doCreateModernProjectStructure ${artifact} ${PWD} ${groupIdFolders}
+
+doMoveQuarkusFilesToModernStructure "${extractedFolderBasePath}/" "${PWD}/${artifact}/"
+
+## cleanup
+rm -r "${extractedFolderBasePath}"
+}
+
+function doMoveQuarkusFilesToModernStructure() {
+## mv dockerfolder can be moved 
+mv "${DEVON_IDE_HOME}/updates/extracted/${artifact}/src/main/docker" "${PWD}/${artifact}/src/main"
+## resoruces folder can be moved
+mv "${DEVON_IDE_HOME}/updates/extracted/${artifact}/src/main/resources" "${PWD}/${artifact}/src/main"
+
+if [ -d "${extractedFolderBasePath}/src/test/resources" ]
+then 
+  mv "${extractedFolderBasePath}/src/test/resources" "${PWD}/${artifact}/src/test/"
+fi
+
+
+##files=`eval find "${extractedFolderBasePath}/*" -type f`
+ServiceFiles=$(grep --with-filename '@Path' -r "${extractedFolderBasePath}/")
+if [[ -n $ServiceFiles ]]
+then 
+  mv ${ServiceFiles/:@Path*} "${PWD}/${artifact}/src/main/java/${groupIdFolders}/${artifact}/service/v1"
+fi
+
+TestFiles=`eval find '${extractedFolderBasePath}/src/test/*' -type f`
+if [[ -n $TestFiles ]]
+then
+  mkdir -p "${PWD}/${artifact}/test/java/${groupIdFolders}/${artifact}/service/v1"
+  mv ${TestFiles} "${PWD}/${artifact}/test/java/${groupIdFolders}/${artifact}/service/v1" 
+fi
+
+projectFiles=`eval find "${extractedFolderBasePath}/" -maxdepth 1 -type f `
+if [[ -n $projectFiles ]]
+then 
+  mv ${projectFiles} "${DEVON_IDE_HOME}/updates/extracted/${artifact}/.mvn" "${PWD}/${artifact}/"
+fi
+
+otherFiles=`eval find "${extractedFolderBasePath}/" -type f`
+if [[ -n $otherFiles ]]
+then 
+  mkdir -p "${PWD}/${artifact}/otherFiles"
+  doEchoAttention "We have found some files provided by Quarkus that couldn't be assigned to our project structure, we moved them into otherFiles folder and you have to relocate them by yourself"
+  mv ${otherFiles} "${PWD}/${artifact}/otherFiles"
+fi
+}
+
+
+#1 artifactId
+#2 path where the project will be created 
+#3 groupId
+function doCreateModernProjectStructure() {
+local projectBasePath="${2}/${1}/src/main"
+mkdir -p "${projectBasePath}" "${projectBasePath}/helm" "${projectBasePath}/java" "${projectBasePath}/java/${3}/general/domain/model" 
+mkdir -p "${projectBasePath}/java/${3}/${1}/domain/repo" "${projectBasePath}/java/${3}/${1}/domain/dao" "${projectBasePath}/java/${3}/${1}/domain/model"
+mkdir -p "${projectBasePath}/java/${3}/${1}/logic" "${projectBasePath}/java/${3}/${1}/service/v1/mapper" "${projectBasePath}/java/${3}/${1}/service/v1/model"
```

Comment:
The `service` package was meanwhile renamed to `rest`. See https://github.com/devonfw/devon4j/blob/master/documentation/guide-structure-modern.asciidoc

Reply:
I also changed `service` to `rest` in doMoveQuarkusFilesToModernStructure.